### ISSUE TITLE
The Dependencies of this packages are becoming outdated.

### ISSUE THREAD

**Issue by nirmalyohannan** [dab246/flutter_super_html_viewer#8]
The dependencies of the packages are becoming outdated and is causing compatibility issue with other packages in the project

**Comment by ooree23** [comment]
Since FLutter 3.32.0, the plugin doesn't work for the web, ui.platformViewRegistry seems to not exist anymore :   
```
lib/src/widgets/web/html_content_viewer_widget_web.dart:261:8: Error: Undefined name 'platformViewRegistry'.
    ui.platformViewRegistry
       ^^^^^^^^^^^^^^^^^^^^
```

**Comment by nirmalyohannan** [comment]
I dont think this package is maintained. Instead try from this github repo [https://github.com/farahfarah1/flutter_super_html_viewer/tree/bugfix/update_flutter](url).  This fork's commit message says it is made compatible with flutter 3.24. This might work.
